### ISSUE TITLE
Ceedling Gcov 0% coverage report

### ISSUE THREAD

**Issue by vatsalnaikgit** [ThrowTheSwitch/Ceedling#642]
I am using Mac OS 12.0.1 , 

`--------------------------
GCOV: OVERALL TEST SUMMARY
--------------------------
TESTED:  14
PASSED:  14
FAILED:   0
IGNORED:  0

---------------------------
GCOV: CODE COVERAGE SUMMARY
---------------------------
test_file.c Lines executed:0.00% of 236
test_file.c Branches executed:0.00% of 398
test_file.c Taken at least once:0.00% of 398
test_file.c No calls
`

**Comment by Letme** [comment]
My first guess would be that you mocked too much :)

**Comment by vatsalnaikgit** [comment]
Haha, I have tested the code in linux vm, and it does work but it just doesn’t show coverage in mac.

**Comment by Letme** [comment]
So there is a difference in how you run the gcov? It has `verbosity` option to get more details

**Comment by vatsalnaikgit** [comment]
Yes, I have recently switched to Mac OS, and not able to run gcov on this.

**Comment by Letme** [comment]
If you cant spot the difference maybe you can paste some logs with `verbosity[4]` before gciv command.

**Comment by MichaelBMiner** [comment]
I am having the same issue [here](https://github.com/gcovr/gcovr/issues/532).

I am running a Jenkins server on Ubuntu 20.04,

**Comment by Letme** [comment]
@MichaelBMiner I do not think you are. Be careful because you need to define `gcov_compiler` and `gcov_linker` and stuff to get --coverage information. What author is saying is that coverage is working on Linux VM, but not on mac. So while this could be gcc difference (maybe mac is using clang to compile) a full verbose log along with `project.yml` would be really helpful in finding something different.

**Comment by Letme** [comment]
But this log also correctly describes the reports in the log file - so I am not sure what you are missing or trying to tell us is wrong. The test fails, so I assume it maybe stopped executing the commands (`utils:gcov` is not run?)

**Comment by Keyur452** [comment]
solution provided under https://github.com/ThrowTheSwitch/Ceedling/issues/447 ticket.

**Comment by vengelhardt** [comment]
@vatsalnaikgit could be closed ?

**Comment by mkarlesky** [comment]
This is quite an old issue now, and it seems to have been a bug in `gcovr` version 4.x itself. The minimum version Ceedling is now known to work with is 5.x, and `gcovr` is rocking along at version 8.x.
